documentation/source/newsfragments/README.rst

```diff
@@ -0,0 +1,50 @@
+:orphan:
```

Comment:
We tried to move documentation to one sentence per line style. How about doing that for this file as well?

Reply:
I would not follow that rule strictly in cases where it still leads to longish lines, but punctuation is in general a good candidate for line breaks.
In the case of this specific file, the ReST is not rendered but read in raw form, so should be pleasant to read in raw as well.